src/views/LoginPage.vue

```diff
@@ -61,56 +122,83 @@ const password = ref("");
 const isLoading = ref(false);
 const userType = ref("patient");
 const userTypeMenu = ref();
+const isRegistering = ref(false);
 
 const router = useRouter();
 const toast = useToast();
 
 const toggleMenu = (event) => {
-    userTypeMenu.value.toggle(event);
+	userTypeMenu.value.toggle(event);
+};
+
+const register = () => {
+	isRegistering.value = !isRegistering.value;
 };
 
 const userTypes = computed(() =>
-    [
-        {
-            label: "Log in as patient",
-            command: () => {
-                userType.value = "patient";
-            },
-        },
-        {
-            label: "Log in as doctor",
-            command: () => {
-                userType.value = "doctor";
-            },
-        },
-        {
-            label: "Log in as admin",
-            command: () => {
-                userType.value = "admin";
-            },
-        },
-    ].filter((item) => !item.label.toLowerCase().includes(userType.value))
+	[
+		{
+			label: "Log in as patient",
+			command: () => {
+				userType.value = "patient";
+			},
+		},
+		{
+			label: "Log in as doctor",
+			command: () => {
+				userType.value = "doctor";
+			},
+		},
+		{
+			label: "Log in as admin",
+			command: () => {
+				userType.value = "admin";
+			},
+		},
+	].filter((item) => !item.label.toLowerCase().includes(userType.value))
```

Comment:
Może moglibyśmy również dodać nagłówek nad polem logowania, który będzie mówił jako który użytkownik się logujemy? Przykładowo "Login as doctor", itp.

Obecnie jedyną informacją, która mówi nam o tym jaki typ logowania wybraliśmy, jest napis "Not a doctor" na dole.

Reply:
Bump

Reply:
Wizualnie jakoś to rozwiązanie do mnie nie przemawia i wymagałoby poprawienia testu. Może w innym miejscu dodać jakiś znaczek/napis sygnalizujący, czy jesteśmy na stronie jako pacjent, doktor czy admin?

Reply:
Podbijam pomysł Przemka, jeśli to źle wygląda to dobrze wymyśleć jakieś inne miejsce ale od strony UX użytkownik może nie wiedzieć co się dzieje trochę (P.S. to normalna rzecz,  że zmiany w kodzie wymagają poprawiania testów i na tym ty polega)

Reply:
Pomyślę nad innym rozwiązaniem, czyli coś innego, niż zmiana tekstu na przycisku, bo do tego nie jestem przekonana.